src/components/FindWallet/LanguageSupportFilter.tsx

```diff
@@ -0,0 +1,149 @@
+      background="background.highlight"
+      borderRadius="base"
+      borderColor="transparent"
+      p={6}
+      zIndex={1}
+    >
+      {({ isExpanded }) => (
+        <>
+          <Heading
+            as="h3"
+            borderBottom={isExpanded ? "1px" : "none"}
+            borderColor="body.light"
+            fontSize="lg"
+            fontWeight={600}
+            lineHeight={1.4}
+            py={1}
+            ps={4}
+            pe={2.5}
+            borderRadius={1}
+            _hover={{ color: "primary.hover" }}
+          >
+            <AccordionButton
+              color="inherit"
+              fontWeight="inherit"
+              fontSize="inherit"
+              p={0}
+              textAlign="initial"
+              _hover={{ background: "transparent" }}
+            >
+              <Box as="span" flex={1}>
+                <Text>{t("page-find-wallet-languages-supported")}</Text>
+              </Box>
+
+              <AccordionIcon
+                color="primary.base"
+                boxSize={9}
+                _hover={{ color: "primary.hover" }}
+              />
+            </AccordionButton>
+          </Heading>
+
+          <AccordionPanel as={List} p={0} m={0}>
+            <Select
+              instanceId="language-support-filter"
+              aria-label={t("page-find-wallet-languages-supported")}
+              options={allWalletsLanguagesOptions}
+              onChange={handleLanguageFilterSelectChange}
+              placeholder={t("page-find-wallet-languages-search-language")}
+              isSearchable
+              variant="outline"
+            />
+
+            <Box mt={2}>
+              <Text color="body.medium" fontSize="sm" lineHeight={1.6}>
+                {t("page-find-wallet-popular-languages")}
+              </Text>
+
+              {getWalletsTopLanguages(5, locale!)
+                .map<ReactNode>((language) => {
+                  return (
+                    <Text
+                      key={language.code}
+                      as="span"
+                      color="primary.base"
+                      fontSize="sm"
+                      lineHeight={1.6}
+                      cursor="pointer"
+                      onClick={() => {
+                        setSupportedLanguage(language.code)
+
+                        trackCustomEvent({
+                          eventCategory: "WalletFilterSidebar",
+                          eventAction: `Language search`,
+                          eventName: getLanguageCodeName(
+                            language.code,
+                            locale!
+                          ),
+                        })
+                      }}
+                    >
+                      {language.langName}
+                    </Text>
+                  )
+                })
+                // `.reduce()` used to generate the comma separator between languages
+                .reduce((prev, curr) => [prev, ", ", curr])}
```

Comment:
nitpick
```suggestion
                .join(", ")
```

Reply:
@pettinarip its an array of objects, doesnt work with `join()`

Reply:
Ah right! good then